dj_crm/data/dj.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+  <record model="dj.compilation" id="dj_crm_base">
```

Comment:
why 2 compilations?

Reply:
one exports the basic settings from res.config.settings the other the "content" with crm.stages and crm.teams i thought it would be nice to be able to export them seperately.

Reply:
but it won't be exported automatically unless you do it

Reply:
@simahawk yes.